src/sentry/integrations/gitlab/search.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.models import Integration
+
+
+class GitlabIssueSearchEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationPermission, )
+
+    def get(self, request, organization, integration_id):
+        try:
+            integration = Integration.objects.get(
+                organizations=organization,
+                id=integration_id,
+                provider='gitlab',
+            )
+        except Integration.DoesNotExist:
+            return Response(status=404)
+
+        field = request.GET.get('field')
+        query = request.GET.get('query')
+        if field is None:
+            return Response({'detail': 'field is a required parameter'}, status=400)
```

Comment:
Is `''` or `0` ok for `field`? If not you might want to use `if not field` instead.

Reply:
I'd assume not. Though in this case it'll be caught by the last line `return Response(status=400)` I don't think the error message makes sense if it's a `0` To me that's still having something in the `field` but just a `0` rather than something valid. Can you pass an empty string in `field` or would nothing come up? Can't remember how that works. 

I would agree that `''` is applicable, but if I put `0` in the field and got `field is a required parameter` then I'd assume something was broken with the api; rather than that value of field is invalid. Which, I can go ahead an add a more specific message at the bottom instead.

Reply:
`''` is a falsey value in python. Because you're reading from request data `''` would come from a URL param like `field=&query=`.